src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -110,11 +110,11 @@ private static void ReleaseElements(IUIAutomationTreeWalker walker, params IList
         /// Get DesktopElements based on Process Id.
         /// </summary>
         /// <param name="pid"></param>
-        /// <param name="dataContext">The context we're workiing with</param>
+        /// <param name="dataContext">The data context</param>
         /// <returns>return null if we fail to get elements by process Id</returns>
         public IEnumerable<DesktopElement> ElementsFromProcessId(int pid, DesktopDataContext dataContext)
```

Comment:
Should this be static now? It feels weird to me that we would have a nonstatic method that could be dealing with UIAutomation objects that don't match

Reply:
Yes it should. I missed that one. Fixed now